zetta_utils/training/lightning/train.py

```diff
@@ -2,21 +2,41 @@
 
 import json
 import os
+from contextlib import ExitStack
+from typing import Final
 
 import pytorch_lightning as pl
 import torch
 import typeguard
 from pytorch_lightning.strategies import ddp
 from pytorch_lightning.utilities.cloud_io import get_filesystem
 
-from zetta_utils import builder, log
+from kubernetes import client as k8s_client  # type: ignore
+from zetta_utils import builder, log, mazepa, parsing
+from zetta_utils.cloud import resource_allocation
+
+DEFAULT_GCP_CLUSTER_NAME: Final = "zutils-x3"
+DEFAULT_GCP_CLUSTER_REGION: Final = "us-east1"
+DEFAULT_GCP_CLUSTER_PROJECT: Final = "zetta-research"
+
+DEFAULT_GCP_CLUSTER: Final = resource_allocation.k8s.ClusterInfo(
+    name=DEFAULT_GCP_CLUSTER_NAME,
+    region=DEFAULT_GCP_CLUSTER_REGION,
+    project=DEFAULT_GCP_CLUSTER_PROJECT,
+)
```

Comment:
These constants are repeated twice, once here and once in `execute_on_gcp_with_sqs`. They should probably move somewhere like `cloud.constants.py`